### DISCUSSION TITLE
API Error when signing transactions

### DISCUSSION THREAD

**Discussion by paulman5** [aptos-labs/aptos-developer-discussions#487] [Dev Doc and Tutorial Feedback]
Hey, I never got this error before since my transaction signing worked just fine, however now I get this error: 
<img width="289" alt="Screenshot 2024-10-13 at 12 35 17" src="https://github.com/user-attachments/assets/de44258a-aa09-4683-8b04-5532ef3d5a5e">

Is this a problem caused on our end or is it something we should resolve in a different way?

**Comment by 0xmihirsahu** [comment]
![image](https://github.com/user-attachments/assets/c24cd7f6-854a-4350-90bb-59c8ad2e80ff)

this is the error


Me and Paul are in the same team.

We need some help with this

**Reply by banool** [reply]
I think you're facing different errors, the original post from @paulman5 is a 400 response, whereas in your case you're getting a 405. Could you share what you're doing to get this error?

**Comment by banool** [comment]
@paulman5 could you share a repro?

**Reply by 0xmihirsahu** [reply]
https://github.com/merkle-builders/FundMate
